swagger.yaml

```diff
@@ -3671,6 +3671,9 @@ definitions:
         x-go-custom-tag: gorm:"type:text"
         type: string
         description: Array of image statuses.
+      domain_name_resolution:
+        type: string
+        description: The domain name resolution for the API.
```

Comment:
The description is a bit too specific - the step can be used to verify any arbitrary domains, it's not specifically for the API (for now, we only use it to verify the API domain, but in the future we might add more things)